gtfs_realtime_translators/translators/swiftly.py

```diff
@@ -4,20 +4,24 @@
 import pendulum
 
 from gtfs_realtime_translators.factories import TripUpdate, FeedMessage
+from gtfs_realtime_translators.validators import RequiredFieldValidator
 
 
-class VtaGtfsRealtimeTranslator:
+class SwiftlyGtfsRealtimeTranslator:
+    # stop_id is not required in constructor as it is being sent as a part of feed but
+    # vta feeds are already configured with stop_id in all environment so remove it when
+    # get a chance to clean up the feed
     def __init__(self, stop_id=None):
-        if stop_id is None:
-            raise ValueError('stop_id is required.')
-
+        RequiredFieldValidator.validate_field_value('stop_id', stop_id)
         self.stop_id = stop_id
 
     def __call__(self, data):
         json_data = json.loads(data)
         entities = []
         for data in json_data["data"]["predictionsData"]:
-            trip_updates = self.__make_trip_updates(data, self.stop_id)
+            stop_id = data.get("stopId", None)
+            RequiredFieldValidator.validate_field_value('stop_id', stop_id)
+            trip_updates = self.__make_trip_updates(data, stop_id)
```

Comment:
Let's just use the original `trip_updates = self.__make_trip_updates(data, self.stop_id)`. No need to extract this field and validate again. 

We can refactor it if we remove the constructor

Reply:
That could be erroneous. URL will have correct stop id. If feed is copied and by mistake, it uses wrong stop id, there will be a mismatch in stop id from constructor and stop id from feed. I would say lets keep it as is. Let me know if you think differently.

Reply:
That's true. Can you make a Jira ticket for removing the constructor and updating the VTA feeds? Then put that ticket # in the comments so we can get it on the backlog.

Reply:
How likely would this come up? @jasonmgoode and @piyushgovani

Reply:
Created new ticket: https://intersection-kb.atlassian.net/browse/TRAN-5325

Reply:
@brian2000 sorry I missed this comment. I think @piyushgovani was saying that we don't need to pass a stop id into the constructor of the translator because the feed endpoint is already stop specific and has the stop_id exposed in the payload.

Reply:
@brian2000 Sorry, even I missed it. @jasonmgoode is right. This PR is not going to touch any existing things. It is just a need of refactoring which was captured in newly created ticket, more like cleanup work. Hope that clarifies.